### DISCUSSION TITLE
2022년 10월 19일

### DISCUSSION THREAD

**Discussion by giscus** [padosum/blog#54] [General]
# 2022년 10월 19일

📚 오늘 도전하고, 배운 것 클린 코드 읽기 아주 유명한 책이다. 계속 계획으론 읽어야지 읽어야지 하다가 이번주에 좀 제대로 읽었다. 경험이 많은 프로그래머들이 개발을 하면서 겪은 것들의 정수가 모아져있었다. 모두 기억하긴 어렵지만 책을 구매했으니 …

https://www.padosum.dev/wiki/2022/10/2022-10-19/

**Comment by Hyeondoonge** [comment]
안녕하세요 연정님!!
공부하다가 집중이 안되어서 놀러왔는데, 블로그보고 감동했습니다 🤩
꾸준하게 TIL 작성하시는 것보고 2차 감동 받고갑니다
자주 놀러올게요

**Reply by padosum** [reply]
반가운 댓글에 기분이 좋아졌네요 ㅎㅎ 저도 자주 놀러가겠습니다 ! 😄
